pkg/daemon/certificate_writer.go

```diff
@@ -190,3 +225,19 @@ func (dn *Daemon) syncControllerConfigHandler(key string) error {
 
 	return nil
 }
+
+func cmToData(cm *corev1.ConfigMap, key string) ([]byte, error) {
+	if bd, bdok := cm.BinaryData["ca-bundle.crt"]; bdok {
+		return bd, nil
+	} else if d, dok := cm.Data["ca-bundle.crt"]; dok {
+		raw, err := base64.StdEncoding.DecodeString(d)
+		if err != nil {
+			// this is actually the result of a bad assumption.  configmap values are not encoded.
+			// After the installer pull merges, this entire attempt to decode can go away.
```

Comment:
which installer pull is this talking about?

Reply:
this is an old comment I must have accidentally moved here. Will delete